File: app-components/src/main/java/org/synsystems/onlypass/components/CrashlyticsModule.java
package org.synsystems.onlypass.components;

import com.crashlytics.android.Crashlytics;

import dagger.Module;
import dagger.Provides;

@Module
public class CrashlyticsModule {
  @Provides
  @AppScope
  public Crashlytics provideCrashlytics() {
    return new Crashlytics();
  }
}

File: app-components/src/main/java/org/synsystems/onlypass/components/CrashlyticsModule.kt
package org.synsystems.onlypass.components

import com.crashlytics.android.Crashlytics

import dagger.Module
import dagger.Provides

@Module
class CrashlyticsModule {

  @Provides
  @AppScope
  fun provideCrashlytics(): Crashlytics = Crashlytics()
}